projects/questionset-editor-library/src/lib/components/header/header.component.html

```diff
@@ -198,12 +199,12 @@
       </div>
   </div>
   <div class="sb-modal-actions">
-    <button id="submitReview" type="submit" class="sb-btn sb-btn-normal" [ngClass]="{'sb-btn-primary': FormControl.form.valid, 'sb-btn-disabled': !FormControl.form.valid}" (click)="buttonEmitter({type: actionType, comment: rejectComment}); showRequestChangesPopup = false;"
-    [disabled]="!FormControl.form.valid" libTelemetryInteract
+    <button id="submitReview" class="sb-btn sb-btn-normal" [ngClass]="{'sb-btn-primary': FormControl.form.valid, 'sb-btn-disabled': !FormControl.form.valid}" (click)="modal.deny('denied')" (click)="saveDraftComments()"
```

Comment:
Please check once if the telemetry is matching with the button clicks.
And one button is having two click events is possible to handle the model.deny from inside the saveDraftComments methods.

Reply:
sure, will check the telemetry event and trigger accordingly

Reply:
Telemetry changed accordingly and two click functions removed.